core/domain/collection_services.py

```diff
@@ -574,6 +576,29 @@ def get_collection_snapshots_metadata(collection_id):
         collection_id, version_nums)
 
 
+def _get_contributor_ids_for_collection(col_id):
```

Comment:
Nope, I don't think so, since at this point no one has actually created a collection yet. But we should fix the incremental update process to ensure there's parity between the two, I think. For now, let's also ignore them for the purposes of the impact score.